### PR TITLE
 Create heat map panel

### PR DESCRIPTION
**Pull request by sumanthreddy542**

### CODE REVIEW

File: src/edu/iastate/metnet/metaomgraph/chart/HeatMapChart.java [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+	private Color textColor;
+	
+	/**
+	 * Constructor
+	 * @param data 2d heatmap data
+	 * @param rowNames row labels
+	 * @param columnNames column labels
+	 * @param transposeData transpose rows and columns?
+	 */
+	public HeatMapChart(double[][] data, String[] rowNames, String[] columnNames, boolean transposeData) {
```
Comment (urmi-21) [inline src/edu/iastate/metnet/metaomgraph/chart/HeatMapChart.java:74]: explain parameters e.g. data dimensions, whar are rowNames and colNames

File: src/edu/iastate/metnet/metaomgraph/chart/HeatMapPanel.java [hunk 2]
```diff
@@ -0,0 +1,881 @@
...
+	private Font textFont;
+	private Color textColor;
+			
+	/**
+	 * Constructor
+	 * @param data heatmap 2d data
+	 * @param rowNames the row labels
+	 * @param colNames the column labels
+	 */
+	public HeatMapPanel(double[][] data, String[] rowNames, String[] colNames) {
```
Comment (urmi-21) [inline src/edu/iastate/metnet/metaomgraph/chart/HeatMapPanel.java:84]: document the parameters better

File: src/edu/iastate/metnet/metaomgraph/chart/HeatMapChart.java (hunk 1 above)
Comment (sumanthreddy542) [reply src/edu/iastate/metnet/metaomgraph/chart/HeatMapChart.java:74]: I think I mentioned the data dimensions as 2d data already.

Comment (urmi-21) [reply src/edu/iastate/metnet/metaomgraph/chart/HeatMapChart.java:74]: I meant to describe what are rows and columns. I think rows are features and columns are samples?

**Review by urmi-21 — APPROVED** [review]

**Pull request merged**